Update AWS config profiles names (#6803)
wazuh/wazuh-documentation: source/cloud-security/amazon/services/prerequisites/considerations.rst

-  ``max_attempts``: The maximum number of attempts including the initial call. This configuration can override the default value set by the retry mode.

You can specify the retry configuration in the ``~/.aws/config`` `configuration file <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-a-configuration-file>`__. The profile section must include the ``max_attempts``, ``retry_mode``, and ``region`` settings.

It is important to use the same profile as the one you chose as your :ref:`authentication method <authentication_method>`. If the authentication method doesn't have a profile, then the ``[default]`` profile must include the configurations. In case the system doesn't have this file, the `aws-s3` Wazuh module defines the following values by default:

-  ``retry_mode=standard``
-  ``max_attempts=10``

> The following example of a ``~/.aws/config`` file sets retry parameters for the *dev* profile:

Reviewer: ```suggestion
The following example of a ``~/.aws/config`` file sets the retry parameters for the *dev* profile:
```

-  ``max_attempts``: The maximum number of attempts including the initial call. This configuration can override the default value set by the retry mode.

You can specify the retry configuration in the ``~/.aws/config`` `configuration file <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-a-configuration-file>`__. The profile section must include the ``max_attempts``, ``retry_mode``, and ``region`` settings.

> It is important to use the same profile as the one you chose as your :ref:`authentication method <authentication_method>`. If the authentication method doesn't have a profile, then the ``[default]`` profile must include the configurations. In case the system doesn't have this file, the `aws-s3` Wazuh module defines the following values by default:

Reviewer 2: ```suggestion
It's important to match this profile section with the :ref:`authentication method profile <aws_profile>`. If the authentication method lacks a profile, then the ``[default]`` profile section must include the configurations. If the configuration file is missing, the `aws-s3` Wazuh module defines the following values by default:
```

   max_attempts=5
   retry_mode=standard

.. note::

>    When using a profile that is not ``[default]`` in a configuration file, the name must include the prefix word *"profile"*.

Reviewer 2: ```suggestion

   When using a profile different to ``default``, ensure the profile name includes the prefix word ``profile``.
```

   dev.proxy.client_cert = /path/to/client_cert.pem
   dev.proxy.use_forwarding_for_https = true

   signature_version = s3v4

.. note::

>    All configurations for ``s3`` and ``proxy`` must start with the profile name.

Reviewer 2: ```suggestion
   All ``s3`` and ``proxy`` configuration sections must start with ``[profile <PROFILE-NAME>]``.
```

> To configure multiple profiles for the integration, declare each profile in the ``~/.aws/config`` file using the same pattern as before. If no profile is declared in the module configuration, the *default* profile will be used.

Reviewer 2: ```suggestion
To configure multiple profiles for the integration, declare each profile section in ``~/.aws/config`` with ``[profile <PROFILE-NAME>]``. If you don't declare a profile section in this configuration file, Wazuh uses the ``default`` profile.
```

   dev.proxy.use_forwarding_for_https = true

>    dev.signature_version = s3v4

Reviewer: ```suggestion
   signature_version = s3v4
```

.. note::

>    The path to the ``s3`` and ``proxy`` sections, as well as ``signature_version``, must start with the profile name.

Reviewer: ```suggestion
   All configurations for the ``s3`` and the ``proxy`` must start with the profile name.
```

> To configure multiple profiles for the integration, declare each profile in the ``~/.aws/config`` file using the same pattern as before. If no profile is declared in the module configuration, the *default* profile is used.

Reviewer: Instead of `is used`. I suggest `will be used` . 

```suggestion
To configure multiple profiles for the integration, declare each profile in the ``~/.aws/config`` file using the same pattern as before. If no profile is declared in the module configuration, the *default* profile will be used.
```
